scripts/lib/setup/installers.sh

```diff
@@ -2,6 +2,16 @@
 
 _localPropertiesPath=./android/local.properties
 
+function getRequiredVersion() {
```

Comment:
I would extract this function to a separate file, maybe. Code deduplication and you can stll use it inside as `$(./get-tool-version.sh conan)`.

Reply:
@mandrigin Updated 07cf219a6153bb1d1a830f25e4ea2145f9246003 to move the method to `packages.sh` and reuse it and `program_exists` from other scripts.